app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cm, first_file, second_file = command.split()
```

Comment:
You code will not work if command will be one of:
"mv"
"mv file1.txt"
"mv file1.txt file2.txt file3.txt"